src/main/java/com/telefonica/iot/cygnus/channels/CygnusFileChannel.java

```diff
@@ -15,7 +15,7 @@
  *
  * For those usages not covered by the GNU Affero General Public License please contact with iot_support at tid dot es
```

Comment:
I'd suggest to use this PR to change also license header, eg. use fiware-cygnus as project name.